<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(223, 223, 223);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);</string>
    </property>
    <widget class="QPushButton" name="pushButton_sair">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
           
	
	
	background-color: rgb(238, 255, 254);
           border: none;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 15px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
          
	
	background-color: rgb(255, 255, 255);
        }</string>
     </property>
     <property name="text">
      <string>Sair</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_redefinir_senha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
           
	
	
	background-color: rgb(238, 255, 254);
           border: none;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 15px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
          
	
	background-color: rgb(255, 255, 255);
        }</string>
     </property>
     <property name="text">
      <string>Redefinir senha</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_usuarios">
        <property name="styleSheet">
         <string notr="true">QPushButton {
           background-color: #4CAF50;
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: #45a049;
        }</string>
        </property>
        <property name="text">
         <string>Usuários</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_restaurantes">
        <property name="styleSheet">
         <string notr="true">QPushButton {
           background-color: #4CAF50;
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: #45a049;
        }</string>
        </property>
        <property name="text">
         <string>Restaurantes </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_hoteis">
        <property name="styleSheet">
         <string notr="true">QPushButton {
           background-color: #4CAF50;
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: #45a049;
        }</string>
        </property>
        <property name="text">
         <string>Hotéis </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_excussoes">
        <property name="styleSheet">
         <string notr="true">QPushButton {
           background-color: #4CAF50;
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: #45a049;
        }</string>
        </property>
        <property name="text">
         <string>Excussões </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ingressos">
        <property name="styleSheet">
         <string notr="true">QPushButton {
           background-color: #4CAF50;
           border: none;
           color: white;
           padding: 10px 20px;
           text-align: center;
           text-decoration: none;
           display: inline-block;
           font-size: 16px;
           margin: 4px 2px;
           cursor: pointer;
           border-radius: 8px;
        }
        QPushButton:hover {
           background-color: #45a049;
        }</string>
        </property>
        <property name="text">
         <string>Ingressos</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
